backends/advanced/src/advanced_omi_backend/routers/modules/knowledge_graph_routes.py

```
@@ -144,6 +156,78 @@ async def get_entity_relationships(
+):
+    """Update an entity's name, details, or icon.
+
+    Also creates entity annotations as a side effect for each changed field.
+    These annotations feed the jargon and entity extraction pipelines.
+    """
+    try:
+        if not request.name and not request.details and not request.icon:
+            raise HTTPException(
+                status_code=400,
+                detail="At least one field (name, details, icon) must be provided",
+            )
```

Ruff (0.14.14)

[warning] Abstract `raise` to an inner function

(TRY301)